src/npm.ts

```diff
@@ -95,6 +95,10 @@ export const npm = {
         command.push("--dry-run");
       }
 
+      if(options.registry){
```

Comment:
Please match code style! In the mean time, I'll work on getting `prettier` integrated

```suggestion
      if (options.registry) {
```